pkg/secrets/clients/k8s/k8s_secrets_client.go

```diff
@@ -6,7 +6,8 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 
-	"github.com/cyberark/secrets-provider-for-k8s/pkg/log"
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/log"
```

Comment:
this can go upwards

Reply:
We don't do that in `provide_conjur_secrets.go`.
Which of these is the better one in your opinion?

Reply:
i would change the one there to
```
import (
	"fmt"
	"strings"

	"github.com/cyberark/conjur-authn-k8s-client/pkg/access_token"
	"gopkg.in/yaml.v2"
	v1 "k8s.io/api/core/v1"

	"github.com/cyberark/secrets-provider-for-k8s/pkg/log"
	"github.com/cyberark/secrets-provider-for-k8s/pkg/log/messages"
	"github.com/cyberark/secrets-provider-for-k8s/pkg/secrets/clients/conjur"
	"github.com/cyberark/secrets-provider-for-k8s/pkg/secrets/clients/k8s"
	"github.com/cyberark/secrets-provider-for-k8s/pkg/secrets/config"
)
```

Reply:
done